2009-01-07  Arthur Loiret  <aloiret@debian.org>

	* sysdeps/unix/sysv/linux/hppa/linuxthreads/malloc-machine.h
	(MALLOC): Adjust __libc_tsd_define arguments.
	(tsd_setspecific, tsd_getspecific): Adjust __libc_tsd_{set,get}
	arguments.

---
 ports/sysdeps/unix/sysv/linux/hppa/linuxthreads/malloc-machine.h |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/ports/sysdeps/unix/sysv/linux/hppa/linuxthreads/malloc-machine.h
+++ b/ports/sysdeps/unix/sysv/linux/hppa/linuxthreads/malloc-machine.h
@@ -63,10 +63,10 @@
 #include <bits/libc-tsd.h>
 
 typedef int tsd_key_t[1];	/* no key data structure, libc magic does it */
-__libc_tsd_define (static, MALLOC)	/* declaration/common definition */
+__libc_tsd_define (static, void *, MALLOC)	/* declaration/common definition */
 #define tsd_key_create(key, destr)	((void) (key))
-#define tsd_setspecific(key, data)	__libc_tsd_set (MALLOC, (data))
-#define tsd_getspecific(key, vptr)	((vptr) = __libc_tsd_get (MALLOC))
+#define tsd_setspecific(key, data)	__libc_tsd_set (void *, MALLOC, (data))
+#define tsd_getspecific(key, vptr)	((vptr) = __libc_tsd_get (void *, MALLOC))
 
 #include <sysdeps/generic/malloc-machine.h>
 
